admin/app/components/solidus_admin/orders/address/component.yml

```diff
@@ -0,0 +1,10 @@
+# Add your component translations here.
+# Use the translation in the example in your template with `t(".hello")`.
+en:
+  save: Save
+  cancel: Cancel
+  back: Back
+  title: "Edit %{type}"
```

Comment:
Also here might be more convenient to split the translations matching the accepted types:

```yaml
title:
  ship: Edit Shipping Address
  bill: Edit Billing Address
```